Documentation/Settings.cfg

```diff
@@ -0,0 +1,17 @@
+[general]
+
+project     = TYPO3 Coding Standards Package
+release     = 0.5.x
+version     = 0.5
+copyright   = MIT license
+Description = A set of coding guidelines for any TYPO3-related project or extension
+
+[html_theme_options]
+
+github_branch        = main
+github_repository    =
```

Comment:
Any reason to not to set the repo url here? Is this just for the edit link?

Reply:
Actually both values aren't used in our case, because one is empty. We could fill the value in, as soon if a `./Documentation/Index.rst` file exists. So in case we advance with documentation we are prepared. Yes, the only reason to have those values is to create the `Edit on GitHub` button. However, at the moment this button doesn't work for projects, like this one, that only have a README file. That's why I left it empty.